bench/wtperf/wtperf.h

```diff
@@ -251,3 +296,14 @@ void	 lprintf(const CONFIG *, int err, uint32_t, const char *, ...)
 __attribute__((format (printf, 4, 5)))
 #endif
 ;
+
+static inline void
```

Comment:
@keithbostic Can you take a look at this function declaration and let me know if this will be inlined properly? The inline semantics confuse me.

No need to look at the rest of the changes.

Thanks,
Alex